device/rsvp/rsvpSessionAttribute.yaml

```diff
@@ -0,0 +1,99 @@
+            It is suggested to include the Local IP, Remote IP, Tunnel ID and LSP ID in the auto-generated Session Name to ensure uniqueness of the name in the test.
+          type: boolean
+          default: true
+        session_name:
+          description: >-
+            If auto_generate_session_name is set to 'false', then the value of this field is used to fill the Session Name field.
+          type: string
+        setup_priority:
+          description: >-
+            Specifies the value of the Setup Priority field. This controls whether the LSP should pre-empt existing 
+            LSP setup with certain Holding Priority if resource limitation is encountered when setting up the LSP. (e.g. bandwidth availability)
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        holding_priority:
+          description: >-
+            Specifies the value of the Holding Priority field. This controls whether a new LSP being created with certain Setup Priority
+            should pre-empt this LSP if resource limitation is encountered when setting up the LSP. (e.g. bandwidth availability)
+          type: integer
+          default: 7
+          minimum: 0
+          maximum: 7
+        local_protection_desired:
+          description: >-
+            This flag permits transit routers to use a local repair mechanism which may result in violation of the explicit
+            route object.  When a fault is detected on an adjacent downstream link or node, a transit router can reroute
+            traffic for fast service restoration.
+          type: boolean
+          default: false
+        label_recording_desired:
+          description: >-
+            This flag indicates that label information should be included when doing a route record.
+          type: boolean
+          default: false
+        se_style_desired:
+          description: >-
+            This flag indicates that the tunnel ingress node may choose to reroute this tunnel without tearing it down.
+            A tunnel egress node SHOULD use the SE Style when responding with a Resv message.
+          type: boolean
+          default: false
+        bandwidth_protection_desired:
+          description: >-
+            This flag indicates to the PLRs along the protected LSP path that a backup path with a bandwidth guarantee is desired.  
+            The bandwidth to be guaranteed is that of the protected LSP, if no FAST_REROUTE object is included in the PATH message;
+            if a FAST_REROUTE object is in the Path message, then the bandwidth specified therein is to be guaranteed.
+          type: boolean
+          default: false
+        node_protection_desired:
+          description: >-
+            This flag indicates to the PLRs along a protected LSP path that a backup path that bypasses at least the next node of the
+            protected LSP is desired.
```

Comment:
This flag indicates to the PLRs along a protected LSP path that, it is desired to have  a backup path that bypasses at least the next node of the  protected LSP.

Reply:
Done.

Reply:
Done.